releasenotes-builder/config/suppressions.xml

```diff
@@ -6,10 +6,13 @@
 
 <suppressions>
     <suppress checks="ClassDataAbstractionCoupling" files="XdocPublisher.java"/>
-    <suppress checks="CyclomaticComplexity" files="Main.java" lines="164"/>
+    <suppress checks="CyclomaticComplexity" files="Main.java" lines="165"/>
     <suppress checks="CyclomaticComplexity" files="CliOptions.java" lines="309"/>
-    <suppress checks="IllegalCatch" files="Main.java" lines="226, 244, 261"/>
+    <suppress checks="IllegalCatch" files="Main.java" lines="230, 248, 265, 282"/>
     <suppress checks="JavadocMethod" files="CliOptions.java"/>
     <suppress checks="ThrowsCount" files="XdocPublisher.java"/>
     <suppress checks="UncommentedMain" files="Main.java"/>
+
+    <!-- Until https://github.com/checkstyle/checkstyle/issues/3590 -->
+    <suppress checks="CustomImportOrder" files="Main.java" lines="33"/>
```

Comment:
I already agreed to use comments suppressions in code in main project, contributions repo could do the same.
But it is not mandatory in this PR.